packages/contracts/foundry_test/WhaleSniper.t.sol

```diff
@@ -161,7 +163,7 @@ contract WhaleSniperPOCTest is eBTCBaseFixture {
         console.log("tcrAfter claim", tcrAfter);
 
         // Now we're in recovery mode so there exist a value such that a liquidation can be triggered willingly by the attacker
-        assertLt(tcrAfter, 1250000000000000000);
+        assertGt(tcrAfter, 1250000000000000000); /// @audit ??? No longer triggering RM?
```

Comment:
curious on reasoning for this change as well

Reply:
This basically asserts that it never triggers RM

 @rayeaster Please see if you can make RM be triggered